doc/changes.md

```diff
@@ -1,5 +1,110 @@
 # Changelog
 
+
+## Release v0.8.0 (2022/12/28)
+
+[![Digital Object Identifier for PyGMT v0.8.0](https://zenodo.org/badge/DOI/10.5281/zenodo.7481934.svg)](https://doi.org/10.5281/zenodo.7481934)
+
+### Highlights
+
+* 🎉 **Eighth minor release of PyGMT** 🎉
+* Added support for tab auto-completion for all GMT default parameters ([#2213](https://github.com/GenericMappingTools/pygmt/pull/2213))
+* Created functions to download GMT remote datasets ([#2200](https://github.com/GenericMappingTools/pygmt/pull/2200))
```

Comment:
Hmm, it looks like #2200 is an internal function, not sure if we want to show this to the public. I thought there was an issue tracking all of the new `load_earth_*` dataset functions done by @willschlitzer but it doesn't look like it...

I'll modify the comment in #1786 to include all the newly added functions, and then we can link to that instead.

```suggestion
* Created functions to download GMT remote datasets ([#1786](https://github.com/GenericMappingTools/pygmt/issues/1786))
```

Reply:
I think this is a good change.